modules/structs/repo.go

```diff
@@ -100,7 +98,7 @@ type EditRepoOption struct {
 	// Either `true` to allow rebase with explicit merge commits (--no-ff), or `false` to prevent rebase with explicit merge commits. `enabled_pull_requests` must be `true`.
 	AllowRebaseMerge *bool `json:"allow_rebase_explicit,omitempty"`
 	// Either `true` to allow squash-merging pull requests, or `false` to prevent squash-merging. `enabled_pull_requests` must be `true`.
-	AllowSquashMerge *bool `json:"allow_squash_merge,omitempty"`
+	AllowSquash *bool `json:"allow_squash_merge,omitempty"`
```

Comment:
Why did you change this?

Reply:
This struct was written by my for this, which initially got put into the go-sdk repo which then was merged into the gitea main library. I wanted all these struct properties to be the same as what they would go to in the models, this one being in conjunction with https://github.com/go-gitea/gitea/blob/master/models/repo_unit.go#L95

Reply:
So the only problem I've got is that you've changed the name of it without changing the json name of it. It's just a bit weird to change one but not the other especially as it's not used anywhere else so won't break API. 

In some ways keeping the merge suffix is reasonable as it's a squash merge whereas squash is part of a rebase so it could be confusing - but, you can't really use a squash in any other way in this context so it shouldn't be confusing.

No problem but you should probably change both.

Reply:
Was keeping the JSON the same as GitHub. Even linked to the GitHub json call in my PR description. Yet internally I prefer all struct names to be the same between API and models.

Reply:
The JSON name makes more sense to a user, especially if they are coming from GitHub's API as I and my org are. The only JSON property I did change to be more similar to the internal naming of Gitea is the "webpage" property which is "homepage" on Github since in Gitea's UI we also call that webpage.